packages/ember/tests/acceptance/sentry-performance-test.js

```diff
@@ -89,45 +78,15 @@ module('Acceptance | Sentry Performance', function (hooks) {
       spans: [
         'ui.ember.transition | route:tracing -> route:slow-loading-route.index',
         'ui.ember.route.before_model | slow-loading-route',
-        'ui.ember.runloop.actions | undefined',
-        'ui.ember.runloop.routerTransitions | undefined',
-        'ui.ember.runloop.render | undefined',
-        'ui.ember.runloop.afterRender | undefined',
-        'ui.ember.runloop.destroy | undefined',
```

Comment:
l: Since you said that the order of these spans is the thing that makes the tests brittle, is there a way we can check that they are still present regardless of the order? Maybe some sort of "just make sure that some runloop spans are recorded" sort of thing? But you're obviously the Ember expert here, so if you say that there's no value in doing that, that's totally fine with me.

Reply:
We could do that, but that also goes into rather brittle territory - e.g. these are all auto-captured, and could even change between versions etc (theoretically).

I guess I can add a check that _at least one_ runloop span has been added, that should be safe to do, and at least ensures that the overall runloop capturing happens!